dirs/dirs.go

```diff
@@ -241,6 +242,7 @@ func SetRootDir(rootdir string) {
 	SnapNamesFile = filepath.Join(SnapCacheDir, "names")
 	SnapSectionsFile = filepath.Join(SnapCacheDir, "sections")
 	SnapCommandsDB = filepath.Join(SnapCacheDir, "commands.db")
+	SnapStoreInfoCacheDir = filepath.Join(SnapCacheDir, "store")
```

Comment:
SnapAuxStoreInfoDir    

also maybe we need a level more   "store"  => "store/aux"  because we might have other store things to cache later